components/openmw-mp/Base/BasePacketProcessor.hpp

```diff
@@ -38,7 +39,7 @@ class BasePacketProcessor
                 throw std::logic_error("processor " + p.second->strPacketID + " already registered. Check " +
                                        processor->className + " and " + p.second->className);
         }
-        processors.insert(typename processors_t::value_type(processor->GetPacketID(), std::move(processor)));
+        processors.emplace(processor->GetPacketID(), std::unique_ptr<Proccessor>(processor));
```

Comment:
"<Proccessor>" looks wrong. Or is there a typo somewhere else in the code?

Reply:
Yes, that's how the template parameter is spelled.

> template<typename Proccessor>